src/App.js

```diff
@@ -1,7 +1,8 @@
-import React from 'react';
-import Form from './Form';
-import Resources from './Resources';
-import Search from './Search';
+import React from "react";
```

Comment:
Many of these changes are just a change of quotes from single to double. You should agree as a team which to use and set that permanently in the `.eslintrc` file (see https://eslint.org/docs/rules/quotes), otherwise you'll just have loads of noise in your commits as you switch back and forth.